src/client/app/components/groups/EditGroupsComponent.tsx

```diff
@@ -209,6 +283,23 @@ class EditGroupsComponent extends React.Component<EditGroupsPropsWithIntl, EditG
 		}
 	}
 
+	private handleGPSChange(e: React.ChangeEvent<HTMLInputElement>) {
+		this.setState({ gpsInput: e.target.value });
+	}
+
+	private handleDisplayChange(e: React.ChangeEvent<HTMLInputElement>) {
+		this.props.editGroupDisplayable(e.target.value === 'true');
+	}
+
+	private handleNoteChange(e: React.ChangeEvent<HTMLTextAreaElement>) {
+		this.props.editGroupNote(e.target.value);
```

Comment:
I'm not sure why but I cannot edit the note. The characters typed do not show but the first one typed shows up when the changes are saved. I tried with an exising and new group and saw it in both cases. The creation of a group works fine.

Reply:
added setstate for both note and display